_posts/2022-02-24-tus-v2.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Towards tus 2.0"
+author: acconut
+redirect_from: /blog/2022/02/23/tus-2/
+---
+
+Seven years ago, in November 2015, [we announced the release of tus 1.0](blog/2015/11/16/tus.1.0.html), the first major and stable version of our uploading protocol. In the past months, we have been actively working on the next steps for tus and wanted to share early details about **tus 2.0**.
+
+tus 2.0 will keep the same principles and approaches for how uploads are performed. This ensures that upgrading the uploading infrastructure from tus 1.0 to 2.0 is as seamless as possible for the end-users as well as for the developers. The most important changes will happen to the structure of the protocol:
+
+The [specification of tus 1.0](/protocols/resumable-upload.html) consists of the core uploading protocol and multiple optional extensions, such as the creation extension, termination extension, and checksum extension. All of these specifications have been hosted on our website [tus.io](/). For **tus 2.0**, we want to make the next step and turn the core uploading protocol into an [IETF](https://www.ietf.org/) Internet Standard. The goal is to encourage native implementations of the uploading protocol in browsers and mobile platform SDKs, as well making resumable uploads accessible to more people. Having an internet standard is beneficial to both objectives. Features that currently exist as optional extensions in tus 1.0 might not be part of the standardized upload protocol. These features will also be included in tus 2.0 in optional extensions, that live outside the standardized upload protocol. This ensures that the standardized upload protocol is concise and focused.
```

Comment:
With my pedant hat on, you might want to rephrase things slightly to something like the following but with your own knowledge and editorial spin on it still.


The [specification of tus 1.0](/protocols/resumable-upload.html) consists of the core uploading protocol and multiple optional extensions, such as the creation extension, termination extension, and checksum extension. All of these specifications have been hosted on our website [tus.io](/) and we have been pleased to see a wide variety of implementations (link?), in part because it is based on open standards like HTTP.

Looking to the future, we'd like to further encourage native implementations of the uploading protocol in browsers and mobile platform SDKs, as well making resumable uploads accessible to more people. To that end,  we would like **tus 2.0**, to be an open standard. In considering how to approach standardisation, we believe that turning the core uploading protocol into a clear and concise standard within the [IETF](https://www.ietf.org/) is most appropriate. We have published a first revision of an Internet-Draft (link) and are engaging with the process of formal adoption into a suitable venue at the IETF.

Features that currently exist as optional extensions in tus 1.0 might not be part of the standardized upload protocol. These features will also be included in tus 2.0 in optional extensions, that live outside the standardized upload protocol. This helps ensure that the standardized upload protocol is concise and focused, while also continuing to support additional use cases.